Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2068,14 +2068,19 @@ void CGContextShowGlyphsWithAdvances(CGContextRef context, const CGGlyph* glyphs
 
 #pragma region Drawing Operations - Basic Shapes
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes only supports the scenario where clipping is not set.
```

Comment:
Also,
* Only works on untransformed coordinates. The cleared region will be cleared in device space.